include/RAJA/pattern/forall.hpp

```diff
@@ -271,11 +301,13 @@ RAJA_INLINE void forall_Icount(ExecutionPolicy&& p,
 
   util::callPreLaunchPlugins(context);
 
-  wrap::forall_Icount(std::forward<ExecutionPolicy>(p),
-                      std::forward<IdxSet>(c),
-                      body);
+  RAJA::resources::EventProxy e = wrap::forall_Icount(r,
+                                  std::forward<ExecutionPolicy>(p),
+                                  std::forward<IdxSet>(c),
+                                  std::forward<LoopBody>(body));
```

Comment:
since this isn't loop_body anymore, you can use std::move(body).

Reply:
Done